cdk/resources/create_config_lambda/config.yaml

```diff
@@ -9,19 +9,14 @@ tornado:
   xsrf_cookie_kwargs:
     samesite: strict
 
+extends:
+  - AWS_SECRETS_MANAGER: ConsoleMeConfigSecret
```

Comment:
Shouldn't be any spaces after the colon (It should be interpreted as a string). In my local testing, I needed the full secret ARN here. Please confirm the same for you and I'll update the docs

Reply:
Great! maybe it would be useful to add some validation with meaningful error, and fix docs obviously.